subsys/usb/usb_descriptor.c

```diff
@@ -112,6 +112,7 @@ struct dev_common_descriptor {
 		struct usb_if_descriptor if0;
 		struct usb_hid_descriptor if0_hid;
 		struct usb_ep_descriptor if0_int_ep;
+		struct usb_ep_descriptor if1_int_ep;
```

Comment:
Both endpoints belong to Interface 0, please rename it to `if0_int_in_ep` and `if0_int_out_ep`. Should not ep_int_out be optional and configurable, as it is not always necessary?

Reply:
Yes, maybe it does make sense to rename CONFIG_HID_OUT_EP_ADDR to CONFIG_HID_INT_OUT_EP_ADDR and CONFIG_HID_INT_EP_ADDR to CONFIG_HID_INT_IN_EP_ADDR

Reply:
ok. I will change and make OUT Endpoint optional